wiki/User_title/zh.md

```diff
@@ -0,0 +1,35 @@
+---
+tags:
+  - 论坛头衔
+  - 等级
+  - 头衔
+  - 用户头衔颜色 # 如果什么时候有了该页面则可以删除这个tag
+  - 用户组颜色
+
+---
+
+# 用户头衔
+
+![](img/dev.png "在Tom94的主页上方可以看到\"osu!dev\" 头衔")
```

Comment:
missing "Screenshot of Tom94's profile information" image alt text, for reference:
```md
![Screenshot of Tom94's profile information](img/dev.png "The \"osu!dev\" title can be seen near the top of Tom94's profile.")
```